lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/configuration/DvaConfiguration.java

```diff
@@ -38,6 +38,8 @@ public class DvaConfiguration {
     // JWS SHA-1 Certificate Thumbprint (Header)
```

Comment:
Were missing a test for this file can we add one now it differs from the DCS equivalent